src/pages/registration/RegistrationFormActions.tsx

```diff
@@ -101,30 +98,37 @@ export const RegistrationFormActions = ({ tabNumber, setTabNumber, validateForm
           </Box>
         )}
 
-        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'start' }}>
+        <Box sx={{ gridArea: 'support-button', display: 'flex', justifyContent: 'space-evenly' }}>
           <Button
+            sx={{
+              borderRadius: '15px 0px 15px 0px',
```

Comment:
```suggestion
              borderRadius: '8px 0px 8px 0px',
```
![bilde](https://github.com/BIBSYSDEV/NVA-Frontend/assets/6313468/3ff4a5a5-0087-4427-aff2-01e8f08bc894)